tests/integration/test_minidump.py

```diff
@@ -422,7 +422,7 @@ def test_minidump_with_processing(
         {
             "id": attachment_id,
             "name": "minidump.dmp",
-            "content_type": "application/octet-stream",
+            "content_type": None,
```

Comment:
Q: Does `sentry` always expect the `content_type` field to exist?

Looking at [this code](https://github.com/getsentry/sentry/blob/42050bfa4f3fcc6b19c9f104eba868dcc8535eaa/src/sentry/attachments/base.py#L58) maybe it does, but I'm not familiar with that part of the codebase. If it's ok to not include the field on null values, let's skip serializing it by adding a `#[serde(skip_serializing_if = "Option::is_none")]` to `ChunkedAttachment.content_type` on `relay-server/src/services/store.rs`.

Reply:
+1, sentry might expect the content type, we should make sure it can handle missing / `null` values.

Reply:
I am also not familiar with the code base. Still, when looking at the handling of attachments processed from envelope items (directly vs. via multipart), a non-existent content type appears to be valid input.

Reply:
Let me check internally and I'll get back to you

Reply:
It seems it's safe for `content_type` to be null or not exist (see [sentry test](https://github.com/getsentry/sentry/blob/6030fda66f27cfb3b4dfee3018a978fb9752a4c1/tests/sentry/api/endpoints/test_event_attachments.py#L28)), and sentry will also [default to `application/octet-stream`](https://github.com/getsentry/sentry/blob/dd0edf1fcc309765d94505f1bac758d7831dcbbf/src/sentry/models/eventattachment.py#L187) if it can't guess the type.

@supervacuus let's skip the serialization when the field is empty as I posted on my [comment above](https://github.com/getsentry/relay/pull/3225#discussion_r1530010648), and I'll merge the PR afterward.